llvm/include/llvm/Cheerp/CheerpWritePass.h

```diff
@@ -0,0 +1,180 @@
+//===-- CheerpWritePass.h - Pass writer for CheerpWriter ------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT for details.
+//
+// Copyright 2011-2023 Leaning Technologies
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/Support/ToolOutputFile.h"
+#include "llvm/Support/FileSystem.h"
+#include "llvm/Support/Path.h"
+#include "llvm/Support/raw_ostream.h"
+#include "llvm/IR/PassManager.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/GlobalValue.h"
+#include "llvm/IR/Type.h"
+#include "llvm/Analysis/LoopInfo.h"
+#include "llvm/Analysis/PostDominators.h"
+#include "llvm/Cheerp/Writer.h"
+#include "llvm/Cheerp/WasmWriter.h"
+#include "llvm/Cheerp/DTSWriter.h"
+#include "llvm/Cheerp/PassRegistry.h"
+#include "llvm/Cheerp/PassUtility.h"
+#include "llvm/Cheerp/I64Lowering.h"
+#include "llvm/Cheerp/SIMDLowering.h"
+#include "llvm/Cheerp/SIMDTransform.h"
+#include "llvm/Cheerp/BitCastLowering.h"
+#include "llvm/Cheerp/JSStringLiteralLowering.h"
+#include "llvm/Cheerp/MemoryInit.h"
+#include "llvm/Transforms/Scalar.h"
+#include "llvm/Transforms/Scalar/EarlyCSE.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Pass.h"
+#include "llvm/Passes/StandardInstrumentations.h"
+#include "llvm/Support/PrettyStackTrace.h"
+#include "llvm/Transforms/Scalar/DCE.h"
+using namespace llvm;
+
+static cl::opt<bool> VerbosePassManager("cheerp-verbose-pm", cl::init(false), cl::Hidden,
```

Comment:
The CI fails when compiling musl with:

`: CommandLine Error: Option 'cheerp-verbose-pm' registered more than once!`

You need to put this in a .cpp file and just declare it here as `extern`. Or better, if possible not include it in the .h at all